mount.go

```diff
@@ -99,6 +100,25 @@ be interacting with the file system.`)
 	}
 	bm := gcsx.NewBucketManager(bucketCfg, storageHandle)
 
+	if mountConfig.StatCacheMaxSizeInMb != config.StatCacheMaxSizeInMbUnsetSentinel {
+		if mountConfig.StatCacheMaxSizeInMb < -1 {
```

Comment:
Do we need to check it here? I believe this should be cached while parsing.

Reply:
This is already checked in mount_config while parsing. Removed from here.